ci/centos7-ci.sh

```diff
@@ -31,9 +32,19 @@ install_dependencies() {
     yum install -y libnghttp2-devel
     install_curl
 
+    yum -y install centos-release-scl
+    yum -y install devtoolset-9 patch wget git make sudo
+    set +eu
+    source scl_source enable devtoolset-9
+    set -eu
+
     # install openresty to make apisix's rpm test work
     yum install -y yum-utils && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    yum install -y openresty-1.21.4.2 openresty-debug-1.21.4.2 openresty-openssl111-debug-devel pcre pcre-devel
+    wget "https://raw.githubusercontent.com/api7/apisix-build-tools/apisix-runtime/${APISIX_RUNTIME}/build-apisix-runtime-debug-centos7.sh"
+    wget "https://raw.githubusercontent.com/api7/apisix-build-tools/apisix-runtime/${APISIX_RUNTIME}/build-apisix-runtime.sh"
+    chmod +x build-apisix-runtime-debug-centos7.sh
+    chmod +x build-apisix-runtime.sh
```

Comment:
Doesn't seem to need `build-apisix-runtime.sh` ？

Reply:
build-apisix-runtime-debug-centos7.sh is called in this script

Reply:
Oh, okay.